packages/epics/src/membership/components/members/member-card.tsx

```diff
@@ -0,0 +1,75 @@
+import { Text } from '@radix-ui/themes';
+import { Card, Badge, StatusBadge, Skeleton } from '@hypha-platform/ui';
+import Image from 'next/image';
+import { SewingPinFilledIcon } from '@radix-ui/react-icons';
+
+type MemberCardProps = {
+  name?: string;
+  surname?: string;
+  nickname?: string;
+  location?: string;
+  avatar?: string;
+  commitment?: number;
+  status?: string;
+  isLoading?: boolean;
+};
+
+export const MemberCard: React.FC<MemberCardProps> = ({
+  name,
+  surname,
+  nickname,
+  location,
+  avatar,
+  commitment,
+  status,
+  isLoading,
+}) => {
+  return (
+    <Card className="w-full h-full p-6 mb-2 flex">
+      {isLoading ? (
+        <Skeleton width={64} height={64} className="rounded-lg mr-3" />
+      ) : (
+        <Image
+          className="rounded-lg mr-3"
+          src={avatar ?? ''}
+          height={64}
+          width={64}
+          alt={nickname ?? ''}
+        ></Image>
+      )}
```

Comment:
The `shadcn/ui` `Skeleton` signature doesn't seem very elegant. I like it more how it works with [radix-ui/theme](https://www.radix-ui.com/themes/docs/components/skeleton#with-children). This was what I had in mind when proposing.

Reply:
The Skeleton component has `loading` and `children` prop:

```jsx
<Flex gap="4">
  <Skeleton loading={true}>
    <Switch defaultChecked />
  </Skeleton>
</Flex>
```

instead of:

```jsx
<Flex gap="4">
  {isLoading ? (
      <Skeleton>
   ):(
      <Switch defaultChecked />
    )
  }
```